strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/storage/validation/groupId/MavenGroupIdLowercaseValidatorTest.java

```diff
@@ -0,0 +1,55 @@
+package org.carlspring.strongbox.storage.validation.groupId;
```

Comment:
It would be better to put this under `org.carlspring.strongbox.storage.validation`. If I recall correctly, this is where all the other Maven validators are.

Reply:
There is no test file under validation package. There is version package and that is where maven test files are added. So I created a groupId package and put mine there. Let me know if you still want me to put it under validation.

Reply:
Okay, I have checked and I see what you mean.
Would you mind sending setting the package name to `org.carlspring.strongbox.storage.validation` in this case?

Reply:
This is the same for the other affected classes that are under this package.

Reply:
Done